pkgs/tools/networking/clatd/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "clatd";
+  version = "1.6";
+
+  src = fetchFromGitHub {
+    owner = "toreanderson";
+    repo = "clatd";
+    rev = "v${version}";
+    hash = "sha256-ZUGWQTXXgATy539NQxkZSvQA7HIWkIPsw1NJrz0xKEg=";
+  };
+
```

the shebang needs to be patched, this does the trick for me, though idk if this is the idiomatic way to package perl stuff

```suggestion

  buildInputs = [
    (perl.withPackages (p: [ p.NetDNS p.NetIP ]))
  ];

```

---

Grepping shows that it is the case I believe.